docs/learn/faq/index.html

```diff
@@ -27,7 +27,8 @@
         &gt; <a href="http://lists.perl.org/">List of mailing lists</a>
     </li>
     <li>
-        &gt; <a href="http://irc.perl.org/">Internet Relay Chat (IRC)</a>
```

Comment:
I suspect nobody merged this because you called the commit "Fix a grammar error or two" and replaced a link to irc.perl.org with a link to a page saying that irc.perl.org is awful. That may or may not be true, but your commit was basically dishonest.